jest.config.json

```diff
@@ -1,5 +1,6 @@
 {
   "verbose": true,
+  "silent": true,
```

Comment:
these two are conflicting. what does it do in the config to define them both?

Reply:
https://github.com/ProjectLibertyLabs/gateway/pull/521#discussion_r1765272223